envbuilder.go

```diff
@@ -540,6 +550,11 @@ func Run(ctx context.Context, options Options) error {
 		closeAfterBuild()
 	}
 
+	if err := remount(); err != nil {
+		// Don't fail at this point as it may be useful to be able to inspect the build.
+		options.Logger(codersdk.LogLevelError, "recreate mountpoint: %w", err)
```

Comment:
Should we error out here?

Reply:
Honestly I'm in two minds about it, but if we fail to remount then it's unlikely that building the image will succeed. Erroring out here is probably better.